src/components/DoctorCard/Shared/Links.js

```diff
@@ -50,7 +50,7 @@ export const ConditionalLink = function ConditionalLink({
   ...props
 }) {
   const link = (
-    <Link href={to} self={self}>
+    <Link href={to} self={self ? true : undefined}>
```

Comment:
Is there any particulari reason to write it like this?

Reply:
For unknown reason I thought `self `is passed to styled component. 
BTW correct way for bool styled prop is:  `self ? 1 :0`.